src/lib/temperature-bounds.ts

```diff
@@ -51,3 +51,25 @@ export const getTemperatureBounds = (
   heat: getHeatBounds(controlBounds),
   cool: getCoolBounds(controlBounds),
 })
+
+export const getSupportedThermostatModes = (
+  device: ThermostatDevice
+): HvacModeSetting[] => {
+  const allModes: HvacModeSetting[] = ['heat', 'cool', 'heat_cool', 'off']
```

Comment:
Can we more this into the API?

Reply:
I think that's a good idea, but is this a blocker for this PR? I'd like to get this in before that, if possible.

Reply:
Not a blocker.